src/main/frontend/components/header.cljs

```diff
@@ -143,6 +149,14 @@
        [:div.repos.hidden.md:block
         (repo/repos-dropdown true)]
 
+       (let [projects (state/sub [:me :projects])]
```

Comment:
To save some spaces in the header, we need to move the external project link to the sub-menu below the user avatar because it's not used that often.